docs/Reference.md

```diff
@@ -253,17 +253,18 @@ Discord utilizes a subset of markdown for rendering message content on its clien
 
 ###### Formats
 
-| Type                    | Structure           | Example                      |
-| ----------------------- | ------------------- | ---------------------------- |
-| User                    | <@USER_ID>          | <@80351110224678912>         |
-| User \*                 | <@!USER_ID>         | <@!80351110224678912>        |
-| Channel                 | <#CHANNEL_ID>       | <#103735883630395392>        |
-| Role                    | <@&ROLE_ID>         | <@&165511591545143296>       |
+| Type                    | Structure           | Example                       |
+| ----------------------- | ------------------- | ----------------------------- |
+| User                    | <@USER_ID>          | <@80351110224678912>          |
+| User \*                 | <@!USER_ID>         | <@!80351110224678912>         |
+| Channel                 | <#CHANNEL_ID>       | <#103735883630395392>         |
+| Role                    | <@&ROLE_ID>         | <@&165511591545143296>        |
+| Application Command     | </NAME:ID>          | </airhorn:816437322781949972> |
```

Comment:
Should probably document the usage with subcommands (and subcommand groups)
`</NAME SUBCOMMAND:ID>`
`</NAME SUBCOMMAND_GROUP SUBCOMMAND:ID>`

Reply:
what about a footer note containing this?
> Subcommands and subcommand groups can also be mentioned by using respectively `</NAME SUBCOMMAND:ID>` and `</NAME SUBCOMMAND_GROUP SUBCOMMAND:ID>`.

Reply:
Sounds good 👍